style.css

```diff
@@ -382,21 +427,355 @@ a.btn2 {
   padding-left: 10px;
 }
 
-.hi {
-  background: #1c1a19;
-  padding-top: 20px;
-  margin-top: -22px;
-  padding-bottom: 30px;
-  text-align: center;
-  padding-right: 50px;
+.foot {
+  justify-content: center;
+  padding: 12px 24px;
 }
 
-.di {
-  text-align: center;
-  font-family: "InaiMathi", sans-serif;
+.aa {
+  flex-wrap: wrap;
+  display: flex;
+  align-items: center;
+}
+
+.aa div {
+  justify-items: center;
+  flex-grow: 1;
+  flex-basis: 325px;
+}
+
+.ab {
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 32px;
+  font-size: 24px;
   line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  display: inline-block;
+}
+
+input {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-bottom: 1px rgb(168, 166, 166) solid;
+  height: 30px;
+}
+
+.iam input {
+  width: 250px;
+}
+
+textarea {
+  font: 1em sans-serif;
+  border: none;
+  background: #fbf8f7;
+  width: 300px;
+  margin-left: -20px;
+}
+
+textarea [type="textarea"]::placeholder {
+  padding-left: 20px;
+  padding-top: 10px;
+}
+
+.bl {
+  background: #ff6b00;
+  padding-bottom: 12px;
+  padding-top: 7px;
+  padding-left: 15px;
+  padding-right: 15px;
   color: #fff;
 }
+
+.bl:hover {
+  background: #fff;
+  color: #ff6b00;
+}
+
+.bl:active {
+  background: #e05304;
+  color: #fff;
+}
+
+.bl:disabled {
+  background: #fff;
+  color: #1c1a19;
+}
+
+.pp {
+  text-align: center;
+}
+
+.pi {
+  margin-top: 150px;
+}
+
+.uu {
+  display: flex;
+  gap: 10px;
+  justify-content: center;
+  padding-right: 35px;
+}
+
+.uu svg path {
+  fill: #1c1a19;
```

Comment:
- [ ] [REQUIRED] Please change the color `#1c1a19;` to correct color `#505F79;` 

![image](https://user-images.githubusercontent.com/63915024/186002501-cabe8511-7b2c-457d-9ae6-c4c7fab41c72.png)

Reply:
Done